packages/ui/src/components/header/header.stories.tsx

```diff
@@ -0,0 +1,173 @@
+import { type Meta, StoryFn, type StoryObj } from '@storybook/react';
+
+import { Button } from '../index.js';
+
+import { Header } from './header.component.js';
+
+const meta: Meta<typeof Header> = {
+  title: 'Components/Header',
+  component: Header,
+  tags: ['autodocs'],
+  decorators: [(Story: StoryFn) => <Story />],
+  parameters: {},
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+/**
+ * > Default usage example
+ */
+export const DefaultStory: Story = {
+  render: ({ brand: _, ...rest }, { globals: { theme } }) => {
+    const brand = theme ? theme.toLowerCase() : 'wbc';
+    return (
+      <>
+        <Header brand={brand === 'btfg' ? 'wbc' : brand} {...rest} />
```

Comment:
just a question, so this example is oriented by the theme provider? just because when u use the controls to change the brand it doesn't change anything.

![image](https://github.com/WestpacGEL/GEL-next/assets/4321993/bab037b7-1b4f-40f9-99d6-2be773928ae6)

Reply:
Yes, I will remove the brand from the table as I did with Footer (which was requested when I did that)